src/main/java/nl/tudelft/lifetiles/graph/view/TileView.java

```diff
@@ -127,12 +162,33 @@ private Color sequenceColor(final Mutation mutation) {
      * @param color
      *            the colour of the vertex
      */
-    private void drawVertex(final String text, final double xcoord,
+
+    private void drawVertex(final SequenceSegment segment, final double xcoord,
```

Comment:
Too many arguments, `xcoord` and `ycoord` could be combined, same goes for `width` and `height`